alchemiscale/storage/statestore.py

```diff
@@ -791,13 +824,15 @@ def queue_taskqueue_tasks(
                   (tq)-[:PERFORMS]->(an:AlchemicalNetwork)
             
             // get the task we want to add to the queue; check that it connects to same network
-            // if it has an EXTENDS relationship, get the task it extends
-            MATCH (tn:Task {{_scoped_key: '{t}'}})-[:PERFORMS*]->(tf:Transformation)<-[:DEPENDS_ON]-(an)
+            // only proceed for cases where task is not already in queue
```

Comment:
Are there any desired behaviour changes here I should be aware of for #80 ?

Reply:
Ah I see you have just changed from != complete to != complete or NULL. Sorry got a bit confused by the diff.

Reply:
Oh yeah, the diff is ugly here. I think for #80 the picture is simplified greatly, so I think we're in good shape there. Will keep that in mind when I review #80 after merging this.